cmd/kops/rollingupdatecluster.go

```diff
@@ -35,11 +37,14 @@ import (
 	"k8s.io/kubernetes/pkg/client/unversioned/clientcmd"
 )
 
+// Command Object for a Rolling Update
 type RollingUpdateOptions struct {
 	Yes       bool
 	Force     bool
 	CloudOnly bool
 
+	ForceDrain      bool
+	FailOnValidate  bool
```

Comment:
Wondering if this is a false positive? Can we make the name a little more verbose here?

```
FailUpdateOnFailedValidation
```

Just so the user is sure about the logic?